packages/core/strapi/lib/Strapi.js

```diff
@@ -157,6 +159,10 @@ class Strapi {
     return this.container.get('auth');
   }
 
+  get sanitizers() {
+    return this.container.get('sanitizers');
+  }
+
```

Comment:
What do you think of adding an alias to get a specific sanitizer here? We're doing the same for services, controllers, etc...

Smth like
```javascript
  get sanitizer(uid) {
    return this.container.get('sanitizers').get(uid);
  }
```